components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -772,6 +773,13 @@ func startAndWatchIDE(ctx context.Context, cfg *Config, ideConfig *IDEConfig, wg
 	// Wait until content ready to launch IDE
 	<-cstate.ContentReady()
 
+	if cfg.isPrebuild() && ideConfig.Prebuild != nil {
```

Comment:
it does not work because terminals never finish in gp validate.

Options
- make gp validate --prebuild actually run headless workspace, this change not needed then, but we need a way to surface tasks then (probably the best way but changing contract of gp validate)
- run IDE prebuild in parallel to user tasks in gp validate 👎 
- change how we wait for user tasks, by adding gp sync-done to each of them together with exit 👎 
- don't run IDE prebuild in non-headless workspace, and add headless mode to gp validate, i.e. see first option (maybe good start, lead to first option later)
-